docs/api.rst

```diff
@@ -330,6 +330,31 @@ Generates Facebook login URL to request access token and permissions.
     fb_login_url = graph.auth_url(app_id, canvas_url, perms)
     print(fb_login_url)
 
+get_code_from_token
+^^^^^^^^^^^^^^^^^^^
+https://developers.facebook.com/docs/facebook-login/access-tokens/expiration-and-extension/
+
+Exchanges an existing long livd user access token on the server side for a
+code that can be redeemed for a long lived user token on the client side.
```

Comment:
"Lived" is spelled wrong.

I don't think this sentence is very clear. This isn't a client-side SDK, so it can be assumed that everything is happening "on the server side".